autogpt_platform/frontend/src/components/analytics/google-analytics.tsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import { useEffect } from "react";
+import Script from "next/script";
+import type { GAParams } from "../../types/google";
```

Comment:
```suggestion
import type { GAParams } from "@/types/google";
```

Reply:
We are keeping the same api so it can be copy pasted